### PR TITLE
Update for inventory unit tests

### PR DESCRIPTION
**Pull request by jbrown-xentity**
Handle exception appropriately for python3

### CODE REVIEW

**Comment by jbrown-xentity** [comment]
Safe to ignore circleci error, that test is defunct.

**Review by FuhuXia — APPROVED** [review]

File: ckanext/usmetadata/plugin/__init__.py [hunk 1]
```diff
@@ -343,14 +343,16 @@ def _default_extras_schema(self):
...
+        try:
+            base_path = base.request.path
+            if base_path == u'/api/3/action/package_create':
+                # This is called when the api is explicitly used
+                for update in local_helper.schema_api_for_create:
+                    schema.update(update)
+            elif base_path == '/api/3/action/resource_create':
+                for update in local_helper.schema_api_for_create:
+                    schema.update(update)
+        except RuntimeError:
```
Comment (nickumia-reisys) [inline ckanext/usmetadata/plugin/__init__.py:355]: I don't understand why this code only needs to run on a RuntimeError

**Review by nickumia-reisys — COMMENTED** [review]
⚠️ I don't think the tests ran on Github Actions either.  All I saw was the database get initialized.

**Comment by jbrown-xentity** [comment]
The tests ran: https://github.com/GSA/ckanext-usmetadata/runs/3735077995#step:7:22

**Comment by nickumia-reisys** [comment]
This is the one from the checks.. https://github.com/GSA/ckanext-usmetadata/runs/3735077995#step:5:20

**Pull request merged**